backend/Makefile

```diff
@@ -8,6 +8,8 @@ PG_STRING = 'host=${PG_HOST} dbname=${PG_DBNAME} user=${PG_USER} password=${PG_P
 # commands
 ## application
 run:
+	GITHUB_CLIENT_ID=${GITHUB_CLIENT_ID} \
+	GITHUB_CLIENT_SECRET=${GITHUB_CLIENT_SECRET} \
```

Comment:
I think the application should get this environment variable from the .env file we should not be setting them in the Makefile as the list env variable can end up being longer than a 2 element list.

Reply:
Good one !
Updated !